tests/test_users/test_services.py

```diff
@@ -19,96 +19,80 @@
     AuthException
 )
 from src.core.pagination.models import PageParams
+from src.core.factories import UserFactory
+from tests.test_users.conftest import DB_USER_SCHEMA
 
 
 def test_register_user_that_already_exists(
     sync_session: Session,
-    register_existing_user_data: UserRegisterSchema,
-    db_users: list[UserOutputSchema]
+    db_user: UserOutputSchema
 ):
     with pytest.raises(AlreadyExists) as exc:
-        register_user(sync_session, register_existing_user_data)
+        register_user(sync_session, DB_USER_SCHEMA)
 
 
 def test_create_user_with_occupied_email(
     sync_session: Session,
-    register_data: dict[str, Any],
-    db_users: list[UserOutputSchema]
+    db_user: UserOutputSchema
 ):
-    user_data = copy.copy(register_data)
-    user_data['email'] = db_users[0].email
+    user_data = UserFactory.build(email=db_user.email, password="testtest", password_repeat="testtest")
     with pytest.raises(AlreadyExists) as exc:
-        register_user(sync_session, UserRegisterSchema(**user_data))
+        register_user(sync_session, user_data)
 
 
 def test_create_user_with_occupied_username(
     sync_session: Session,
-    register_data: dict[str, Any],
-    db_users: list[UserOutputSchema]
+    db_user: UserOutputSchema
 ):
-    user_data = copy.copy(register_data)
-    user_data['username'] = db_users[0].username
+    user_data = UserFactory.build(username=db_user.username, password="testtest", password_repeat="testtest")
     with pytest.raises(AlreadyExists) as exc:
-        register_user(sync_session, UserRegisterSchema(**user_data))
+        register_user(sync_session, user_data)
 
 
 def test_if_only_one_user_was_returned(
     sync_session: Session,
-    db_users: list[UserOutputSchema]
+    db_user: UserOutputSchema
 ):
-    user = get_single_user(sync_session, db_users[1].id)
+    user = get_single_user(sync_session, db_user.id)
 
-    assert user.id == db_users[1].id
+    assert user.id == db_user.id
 
 def test_raise_exception_while_getting_nonexistent_user(
     sync_session: Session,
-    db_users: list[UserOutputSchema]
+    db_user: UserOutputSchema
 ):
     with pytest.raises(DoesNotExist) as exc:
-        get_single_user(sync_session, len(db_users)+2)
+        get_single_user(sync_session, len([db_user])+2)
 
 
 def test_if_multiple_users_were_returned(
     sync_session: Session,
-    db_users: list[UserOutputSchema]
+    db_user: UserOutputSchema
 ):
     users = get_all_users(sync_session, PageParams(page=1, size=5))
-    assert len(users.results) == len(db_users)
+    assert users.total == len([db_user])
```

Comment:
```suggestion
    assert users.total == 1
```
skoro i tak wartosc po prawej zawsze jest stala i rowna 1